lampod-cli/src/args.rs

```diff
@@ -150,7 +150,7 @@ pub fn parse_args() -> Result<LampoCliArgs, lexopt::Error> {
                 bitcoind_pass = Some(var);
             }
             Long("restore-wallet") => {
-                restore_wallet_flag = true;
+                mnemonic = parse_mnemonic();
```

Comment:
Still did not change my review, this code need to be done inside the main.rs code

The arg parsing should not do any magic, keep the boolean value and handle it inside the main.rs, please

Reply:
Will do it soon! Exams going on :)